### ISSUE TITLE
Libreoffice fails to launch

### ISSUE THREAD

**Issue by SolitudeSF** [void-linux/void-packages#9231]
Fails to launch with error message:
```
LibreOffice 6.2 - Fatal Error: The application cannot be started. 
The user interface language cannot be determined.
```
appimage or version from arch launches fine.

**Comment by newbluemoon** [comment]
No problems here on fully updated x86_64-musl. Cannot test glibc right now, but there has been a glibc-locales update. Maybe something went wrong? Is your locale set up correctly? Did you try to reconfigure/reinstall forcefully and check your `/etc/default/libc-locales`.

**Comment by SolitudeSF** [comment]
`everything seems normal. reinstalled glibc-locales just to be sure. same error.`

**Comment by roswell-1947** [comment]
No problems here on fully updated x86_64. I did reconfigure glibc-locales after the update though.

**Comment by pullmoll** [comment]
Please try to install `openjdk-jre` and see if the problem goes away.
If it doesn't, please try to create a temporary user and see if libreoffice starts when it creates a new configuration. Alternatively backup your `$HOME/.config/libreoffice`, remove it and try again.

FWIW libreoffice works fine here with `LANG=de_DE.UTF-8`.

**Comment by SolitudeSF** [comment]
doesnt work from new user, with java installed and configuration cleared.

**Comment by SolitudeSF** [comment]
okay, i get it now. i got bamboozled by package splitting. i thought just installing `libreoffice-writer` was enough. it launches after i installed entire `libreoffice` package. i guess thats because it pulls language packs. maybe they should be added as deps to `libreoffice-*` too?

**Comment by pullmoll** [comment]
The split was done to *not* automagically have some basic language support installed but just the subpackages you want and their required dependencies. The average user who doesn't care will probably install libreoffice and be fine. I don't think it makes sense to reintroduce a dependency on libreoffice-i18n-en-US now.

**Comment by hfel** [comment]
Hi! I just encountered the same problem -- how about putting a message at the end of the install process informing the user they need to install a language pack?

**Comment by Forevera-Noob** [comment]
Hi SolitudeSF,

I just encountered the same problem, but in Linux Mint 21.1 Vera.

I updated to Vera and LibreOffice didn't work anymore.
I de-installed it and installed it again based on https://www.linuxcapable.com/how-to-install-libreoffice-on-linux-mint/

Then I got the following error message:

`LibreOffice 7.5 - Fatal Error: The application cannot be started. The user interface language cannot be determined.`

Your comment of March 2nd 2019 helped me, but libreoffice-i18n-en-US doesn't exit anymore.
So someone helped me and installed **libreoffice-l10n-en-gb**

`sudo apt install libreoffice-l10n-en-gb`

Now it works again.

I thought I'd add this solution. Thanx for you original question and solution.
